lib/cocoapods/installer/user_project_integrator/target_integrator.rb

```diff
@@ -30,6 +30,7 @@ def integrate!
             # TODO: refactor into Xcodeproj https://github.com/CocoaPods/Xcodeproj/issues/202
             project_is_dirty = [
               XCConfigIntegrator.integrate(target, native_targets),
+              add_pods_resources,
```

Comment:
@kylef that's actually the opposite, but good point
`add_pods_resources` will only be necessary when doing static library integration, so I'll have to cleanup those resources if we are doing dynamic framework integration but those resources were installed during a previous static libs integration.